src/main/java/app/controller/HomeController.java

```diff
@@ -21,4 +25,18 @@ public String register(){
         return "views/auth/register";
     }
 
+    @GetMapping("/detail-layout")
```

Comment:
Action này chỉ để test thôi nhỉ, đúng thế thì to trực tiếp chị, để chị mợt pull nhé